### ISSUE TITLE
how to train from the last check point?

### ISSUE THREAD

**Issue by justanotherYO** [YunYang1994/tensorflow-yolov3#172]
I trained from scratch for 30 epoch and got the test loss around ~16. Now I wish to train again but with a smaller learning rate. What should I do to resume training? There are several saved model in checkpoint folder, and I wish the training restart from the last checkpoint saved before. Thx!

**Comment by YunYang1994** [comment]
__C.TRAIN.INITIAL_WEIGHT        = "your_checkpoint_file"

**Comment by jersonal** [comment]
> __C.TRAIN.INITIAL_WEIGHT = "your_checkpoint_file"

您好，我尝试过更改INITAL_WEIGHT，但是再次训练的时候Loss值依然跟第一次训练的loss差不多。
第一次300 epoch结束
Restoring weights from: /mnt/aic/train/model/yolov3_train_loss=7.59.ckpt-300
再次训练从301 epoch开始。
Epoch: 301 Time: 2020-05-26 16:51:49 Train loss: 2090.81  Saving: yolov3_train_loss=2090.81.ckpt
